src/worker/exchanges/whitebit.ts

```diff
@@ -0,0 +1,108 @@
+import Exchange from '../exchange'
+
+export default class WHITEBIT extends Exchange {
+  id = 'WHITEBIT'
+
+  protected endpoints = {
+    PRODUCTS: [
+        'https://whitebit.com/api/v4/public/collateral/markets',
```

Comment:
why collateral/markets

Spot Endpoint is /markets

<img width="1275" height="728" alt="image" src="https://github.com/user-attachments/assets/522e68b3-8b96-4c6a-970e-8fa04ddcbd51" />

Reply:
Looks like I missed the `type` property